editor/block-mover/test/mover-label.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { getBlockMoverLabel, getMultiBlockMoverLabel } from '../mover-label';
+
+describe( 'block mover', () => {
+	const dir_up = -1,
```

Comment:
My worry is: Why is this relevant to the tests only, and not runtime usage?

The alternative I see is exporting constants from `mover-label.js`:

```js
export const DIRECTION_UP = -1;
export const DIRECTION_DOWN = 1;
```

_(Realizing now that JavaScript standards have no equivalent to [PHP's upper-case constants](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#naming-conventions))_

These constants could be imported both in tests, but also in `<BlockMover />`, and can be revised once and apply everywhere.

But still, I think it might be overkill. I'm not really requesting any changes here, just trying to understand why we'd care to special-case the tests.